src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs

```diff
@@ -40,7 +40,7 @@ public static TeamNameViewModel Map(TeamTournamentDto team)
             {
                 Id = team.TeamId,
                 Name = team.TeamName,
-                DivisionName = $"Division {team.DivisionId}"
+                DivisionName = team.DivisionName
```

Comment:
You should not have removed checklist 😄 

- Please update corresponding UT

Reply:
UT it is a Unit Test , as I understand?

Reply:
yes)